<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>NextPolygon</class>
 <widget class="QDialog" name="NextPolygon" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Next Polygon</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QWidget" native="1" name="widget_2" >
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>9</number>
            </property>
            <item>
             <widget class="QLabel" name="label_1" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text" >
               <string>Pennies:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_1" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="minimumSize" >
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" >
           <item>
            <widget class="QLabel" name="label" >
             <property name="font" >
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Pennies Left:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pennies_left_label" >
             <property name="minimumSize" >
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="habitat_combo_label" >
            <property name="minimumSize" >
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Habitat Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="habitat_combo" >
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text" >
              <string/>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Hard Bottom</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>New Item</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Soft/Sandy</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Mixed</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>88</width>
         <height>89</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pbnShapeFinished" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Finished With Fishery</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnShapeDiscard" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Discard Last Shape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnMoreShapes" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>More Shapes This Fishery</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>widget</zorder>
  <zorder>label</zorder>
  <zorder>pennies_left_label</zorder>
  <zorder>pennies_left_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
